src/test/javascript/grouping.controller.test.js

```diff
@@ -1812,7 +1812,12 @@ describe("GroupingController", () => {
             scope.pageSelected = true;
             scope.membersInCheckboxList = {"testiwta": true, "testiwtb": true, "testiwtc": true};
             scope.cancelRemoveModal();
-            expect(scope.membersInCheckboxList).toEqual({});
+            for (const mem in scope.membersInCheckboxList) {
+                //codacy will complain without if statement
+                if (typeof scope.membersInCheckboxList[String(mem)] !== "undefined") {
+                    expect(scope.membersInCheckboxList[String(mem)]).toBeFalse();
+                }
+            }
```

Comment:
The .forEach function for an array might work to get around the Codacy issue.

Reply:
This function also shows as an undefined function. The recent push doesn't address this and the previous comment. I can do a `for (i = 0; i < scope.membersInCheckboxList.length; i++)` loop but that will probably still through a Codacy complaint since I'm using the `i` as I would with `mem`. The complaint is that we want to check the body of the `for` loop so that no "unwanted behavior" is performed.